platform/security/core/security-core-services/src/main/java/ddf/security/service/impl/SubjectUtils.java

```diff
@@ -47,69 +48,18 @@
 import org.slf4j.LoggerFactory;
 
 /** Utility class used to perform operations on Subjects. */
-public final class SubjectUtils {
-
-  public static final String GUEST_DISPLAY_NAME = "Guest";
+public final class SubjectUtils implements SubjectOperations {
```

Comment:
❓ Why not use the *Impl naming convention like the others?

Reply:
I was torn between making it an impl or leaving it with the name everyone is used to. I didn't want to make SubjectUtils and SubjectUtilsImpl like I did with the others, because I hate "utils"